index.js

```diff
@@ -2,7 +2,6 @@ var pjson = require('./package.json')
 
 exports.name = 'agol'
 exports.hosts = true
-exports.pattern = '/:id/:item'
```

Comment:
1. true
2. alright

we should probably either flesh out the pattern idea and document it or remove it from providers